application/models/QuestionTheme.php

```diff
@@ -837,9 +837,11 @@ public static function getQuestionXMLPathForBaseType($type)
     {
         static $questionThemeCache = [];
```

Comment:
I dislike to use Cache for Static var … Static is  not caching ;)

Reply:
yeah makes sense, cant use it everywhere php will run out of memory eventually when we use it to much. Will look at yii's caching

Reply:
Currently : it's a static var :). I don't ask to put it in cache (maybe in future), just more clear name :)

Reply:
I also propose the creation of a `protected static $questionThemeCache = [];` data member to make the file more readable and the cache more accessible.

Reply:
`protected static $questionThemeStatic = []` ;)